components/Layout.js

```diff
@@ -12,11 +15,55 @@ export const Layout = ({ children }) => (
 
     <footer className="mt-20 w-full bg-dark-300 text-white">
       <div className="max-w-screen-xl mx-auto p-6 grid grid-cols-[2fr_1fr] lg:grid-cols-[3fr_1fr]">
-        <nav>menu</nav>
-        <div>weather</div>
+        <nav>
+          <ul className="flex flex-col lg:flex-row">
+            <li className="lg:pr-3">
+              <Link href="/home">
+                <a className="underline hover:no-underline text-accent-200 decoration-accent-200 block mb-3">
+                  Hlavní stránka
+                </a>
+              </Link>
+            </li>
+            <li className="lg:px-3">
+              <Link href="/home">
+                <a className="underline hover:no-underline text-accent-200 decoration-accent-200 block mb-3">
+                  Vyhledávání
+                </a>
+              </Link>
+            </li>
+            <li className="lg:px-3">
+              <Link href="/home">
+                <a className="underline hover:no-underline text-accent-200 decoration-accent-200 block mb-3">
+                  Přidat článek
+                </a>
+              </Link>
+            </li>
+            <li className="lg:px-3">
+              <Link href="/home">
+                <a className="underline hover:no-underline text-accent-200 decoration-accent-200 block mb-3">
+                  Váš profil
+                </a>
+              </Link>
+            </li>
+            <li className="lg:px-3">
+              <Link href="/home">
+                <a className="underline hover:no-underline text-accent-200 decoration-accent-200 block mb-3">
+                  Odhlásit se
+                </a>
+              </Link>
+            </li>
```

Comment:
const menuItems = ['Hlavní stránka', .......]

and then inside JSX

```
{menuItems.map((item) => (
 <li className="lg:px-3">
              <Link href="/home">
                <a className="underline hover:no-underline text-accent-200 decoration-accent-200 block mb-3">
                  {item}
                </a>
              </Link>
            </li>
            ))}
```

the menuItems could be ultimately reused in other places (if they will have the same content)